supernovacontroller/sequential/i3c.py

```diff
@@ -913,14 +917,17 @@ def ccc_broadcast_enec(self):
         # Note: The command name 'ccc_broadcast_ENEC' should be handled appropriately in _process_response
         return self._process_response("ccc_broadcast_enec", responses)
 
-    def ccc_broadcast_disec(self):
+    def ccc_broadcast_disec(self, events: list):
         """
         Performs a broadcast DISEC (Disable Events Command) operation on the I3C bus.
 
         This method sends a broadcast command to disable events on all devices on the I3C bus.
         The operation's success status is checked, and it returns a tuple indicating whether the operation
         was successful along with the relevant data or error message.
 
+        Args:
+        events: A list containing events to be disabled.
```

Comment:
Please, mention that the type of element of the list belongs to the enum DISEC: ``DISEC.DISINT``, ``DISEC.DISCR``, ``DISEC.DISHJ``. For instance [DISEC.DISINT, DISEC.DISHJ]